can/interfaces/ixxat/canlib.py

```diff
@@ -140,8 +143,18 @@ def _recv_internal(self, timeout):
     def send(self, msg: Message, timeout: Optional[float] = None) -> None:
         return self.bus.send(msg, timeout)
 
-    def _send_periodic_internal(self, msg, period, duration=None):
+    def _send_periodic_internal(self, msgs, period, duration=None):
```

Comment:
Good change! Please also rename `msg` in the next line and the two receivers, to really restore consistence here